test/policies/terminate/terminate.test.js

```diff
@@ -0,0 +1,45 @@
+const testHelper = require('../../common/routing.helper');
+const Config = require('../../../lib/config/config');
+const helper = testHelper();
+
+describe('@terminate', () => {
```

Comment:
Is there a better word for this than `terminate`?  Sounds like we're killing something with fire.  We're just short-circuiting the pipeline to return a response.  Thoughts?

/cc @altsang

Reply:
there is a broader possible policy - `response` 
That can return any kind of content-type, for example JSON 
I would imagine this policy would allow some JS execution. Kind of variant of Expression wrapped as `res.json("whatever")`

However it is overkill fo this use case